examples/sentence-transformers-training/paraphrases/training_paraphrases.py

```diff
@@ -0,0 +1,149 @@
+"""
+Note: This script was modified with the v3 release of Sentence Transformers.
+As a result, it does not produce exactly the same behaviour as the original script.
+"""
+
+import logging
+import traceback
+from datetime import datetime
+
+from datasets import load_dataset
+from sentence_transformers import SentenceTransformer
+from sentence_transformers.evaluation import EmbeddingSimilarityEvaluator
+from sentence_transformers.losses import MultipleNegativesRankingLoss
+from sentence_transformers.similarity_functions import SimilarityFunction
+from sentence_transformers.trainer import SentenceTransformerTrainer
+from sentence_transformers.training_args import (
+    BatchSamplers,
+    MultiDatasetBatchSamplers,
+    SentenceTransformerTrainingArguments,
+)
+
+from optimum.habana import GaudiConfig, GaudiTrainer, GaudiTrainingArguments, SentenceTransformerGaudiTrainer
+from optimum.habana import SentenceTransformerGaudiTrainingArguments
+
+from optimum.habana.sentence_transformers.modeling_utils import adapt_sentence_transformers_to_gaudi
+adapt_sentence_transformers_to_gaudi()
+
+
+
+# Set the log level to INFO to get more information
+logging.basicConfig(format="%(asctime)s - %(message)s", datefmt="%Y-%m-%d %H:%M:%S", level=logging.INFO)
+
+model_name = "distilroberta-base"  
```

Comment:
Can you update this to take other models as command line argument? eg. When running with miniLM

Reply:
done